components/cdc/tests/integrations/test_cdc.rs

```diff
@@ -220,8 +220,8 @@ fn test_cdc_rawkv_basic() {
         )))
         .unwrap();
 
-    // If tikv enable ApiV2, raw key needs to start with 'r';
-    let (k, v) = (b"rkey1".to_vec(), b"value".to_vec());
+    // If tikv enable ApiV2, raw key needs to start with 'r' + keyspace_id;
+    let (k, v) = (b"r\x00\x00\x00key1".to_vec(), b"value".to_vec());
```

Comment:
Test keys starting with 'r' or 'x' are not only in one place, please check other test cases.

Reply:
All tests of the CDC module are passed.